<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>595</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Traffic Police</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#form{
background:url(img/pbg4.jpg)
}

QLabel{
color: white;
}

#forgot_password{
background:transparent;
color:white;
}
#forgot_password:hover{
border-radius:8px;
border: 1px solid red;
color:white;
background: transparent;
}
#show_password{
border-radius:8px;
background:grey;
color:white;
}
#show_password:hover{
border-radius:8px;
border: 1px solid red;
color:white;
background:grey;
}

#back{
background:transparent;
border-radius:8px;
border: 1px solid white;
color:white;
}
#back:hover{
background:transparent;
border-radius:8px;
border: 1px solid red;
color:white;
}

#registration{
background:transparent;
color:white;
}
#registration:hover{
border-radius:8px;
border: 1px solid red;
color:white;
background: transparent;
}

#enter_background{
background: rgba(0,0,0,0.7);
border-radius:15px;
}
#registration_frame{
background: rgba(0,0,0,0.7);
border-radius:15px;
}
#icon{
background:#2312ff;
border-radius: 60px;
}

QLineEdit{
background:transparent;
border:2px solid #2312ff;
border-radius:15px;
color:white;
}
QLineEdit:focus{
background:transparent;
border:2px solid red;
border-radius:15px;
color:white;
}
#enter{
border-radius:15px;
color:white;
border:2px solid #2312ff;
background:transparent;
}
#enter:hover{
border-radius:15px;
border:2px solid red;
color:white;
background: transparent;
}
#title{
color:white;
border:4px solid blue;
border-radius:15px;
}
#employee{
background:transparent;
border:2px solid #2312ff;
border-radius:15px;
color:white;
}
#citizen{
background:transparent;
border:2px solid#2312ff;
border-radius:15px;
color:white;
}
#employee:hover{
border-radius:15px;
border:2px solid red;
color:white;
background: transparent;
}
#citizen:hover{
border-radius:15px;
border:2px solid red;
color:white;
background: transparent;
}
#employee:disabled{
border-radius:15px;
border:2px solid red;
color:white;
background: transparent;
}
#citizen:disabled{
border-radius:15px;
border:2px solid red;
color:white;
background: transparent;
}





</string>
  </property>
  <widget class="QWidget" name="form">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>771</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>20</number>
    </property>
    <property name="text">
     <string>Traffic Police App</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="enter_background">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>371</width>
      <height>381</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="password_line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Пароль</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enter">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>310</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Войти</string>
     </property>
    </widget>
    <widget class="QPushButton" name="employee">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>для служащих</string>
     </property>
    </widget>
    <widget class="QPushButton" name="citizen">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>для граждан</string>
     </property>
    </widget>
    <widget class="QPushButton" name="forgot_password">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Забыли пароль?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registration">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Регистрация</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
    <widget class="QPushButton" name="show_password">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>220</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="username">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Имя пользователя</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="phone">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Номер телефона</string>
     </property>
    </widget>
    <widget class="QLabel" name="message">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="icon">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/img/question.png</normaloff>:/img/question.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
